msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -65,11 +67,14 @@ Content-Type: application/json
   "@microsoft.graph.teamCreationMode": "migration",
   "template@odata.bind": "https://graph.microsoft.com/beta/teamsTemplates('standard')",
   "displayName": "My Sample Team",
-  "description": "My Sample Team’s Description",
-  "createdDateTime": "2020-03-14T11:22:17.067Z"
+  "description": "My Sample Team’s Description"
+  "createdDateTime": "2020-03-14T11:22:17.043B"
```

Comment:
don't append any random letter for `createdDateTime`, keep the upper case Z. The literal "Z" is actually part of the ISO 8601 datetime standard for UTC times